src/main/java/duke/command/FindTask.java

```diff
@@ -0,0 +1,28 @@
+package duke.command;
+
+import duke.task.DukeException;
+import duke.task.Task;
+
+import java.util.ArrayList;
+
+public class FindTask extends Command {
+    
+    public FindTask(ArrayList<Task> tasks) {
+        super(tasks);
+    }
+
+    public void findTask(String [] arr) {
+        String filterWord;
+        try {
+            filterWord = parser.getFilterWord(arr);
+            ArrayList<Task> shallowCopy = new ArrayList<>(tasks);
```

Comment:
A plural form name could be used instead for names representing a collection of objects